### PR TITLE
Add missing metadata to subcharts

### PR DESCRIPTION
**Pull request by kfox1111**

### CODE REVIEW

File: charts/spire/charts/spiffe-csi-driver/Chart.yaml [hunk 1]
```diff
@@ -4,3 +4,18 @@ description: A Helm chart to install the SPIFFE CSI driver.
 type: application
 version: 0.1.0
 appVersion: "0.2.3"
+keywords: ["spiffe", "csi-driver"]
+home: https://github.com/spiffe/helm-charts/tree/main/charts/spire
```
Comment (faisal-memon) [inline charts/spire/charts/spiffe-csi-driver/Chart.yaml:8]: Will these URLS need to change once we get everything re organized?

**Review by marcofranssen — APPROVED** [review]
LGTM

File: charts/spire/charts/spiffe-csi-driver/Chart.yaml (hunk 1 above)
Comment (kfox1111) [reply charts/spire/charts/spiffe-csi-driver/Chart.yaml:8]: hmm... true. should I just make them point at the root of the repo, or leave them as is and we can change it when we break them out?

Comment (faisal-memon) [reply charts/spire/charts/spiffe-csi-driver/Chart.yaml:8]: Leave as is. I was just confirming.

**Review by faisal-memon — APPROVED** [review]

**Pull request merged**